openpitrix/values.yaml

```diff
@@ -47,6 +55,32 @@ global_config:
   max_repo_events: 20
 
 
+notification:
+  version: latest
+  image: openpitrix/notification:latest
+  flyway_image: openpitrix/notification:flyway
+  service_port: 9200
+  backend_port: 9200
```

Comment:
notification port is 9201 in openpitrix

```
func NewClient() (*Client, error) {
	conn, err := manager.NewClient(constants.NotificationHost, constants.NotificationPort)
	if err != nil {
		return nil, err
	}
	return &Client{
		NotificationClient: nfpb.NewNotificationClient(conn),
	}, nil
}
```
backend_port should not be changed by user.